src/data_storage/sqlalchemy/models.py

```diff
@@ -30,21 +31,24 @@ def get_connection_engine():
 class Story(Base):
     __tablename__ = "story"
 
-    id = Column(Integer, primary_key=True)
+    id = Column(String(36), primary_key=True)
     name = Column(String(200), nullable=False)
 
 
 class Node(Base):
+    """Make boolean flag to show root node or name always root node as Root"""
+
     __tablename__ = "node"
 
     id = Column(Integer, primary_key=True)
     story = Column(
-        Integer,
+        String(36),
```

Comment:
Can use https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#sqlalchemy.dialects.postgresql.UUID

Reply:
I suppose that we can use not only uuid for id key. It related to DynamoDB that id may be usual string